src/homework2/task3.py

```diff
@@ -14,7 +14,12 @@ def sub_string(str_):
     """
 
     # write your code here
-    return ''  # write return value here
+    new_str_ = ""
+    marks = ['.', ',', '!', '?', '-', ':', '(', ')', '"', '\'', ';', ' ']
```

Comment:
redundant checks. we shouldn't delete punctuation marks here

Reply:
fixed